samples/matter/template/sysbuild/mcuboot/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -16,7 +16,6 @@ CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y
 
 # The following configurations are required to support simultaneous multi image update
 CONFIG_PCD_APP=y
```

Comment:
pcd, upgrade only, flash simulator, zcbor - not needed in these files

Reply:
Removed all entries from nrf7002dk files. It should be OK now.